src/components/Form/Field.tsx

```diff
@@ -112,7 +118,28 @@ const ArrowDown = ({ size, fill, ...props }) => (
   </svg>
 )
 
-const Field = React.forwardRef(
+const Field = React.forwardRef<
+  HTMLInputElement,
+  {
+    value?: string | number
+    onChange?: (
+      event: InputEvent,
+      value: string | number,
+      shouldValidate: boolean
+    ) => void
+    name?: string
+    autoComplete?: boolean
+    type?: string
+    label?: string
+    disabled?: boolean
+    error?: string | boolean
+    onInc?: () => void
+    onDec?: () => void
+    icon?: IconType
+    simulate?: string
+    renderInput: React.FC<Record<string, unknown>>
```

Comment:
Looks good for now, but I believe we should specify an interface to define what props are available to an input. If we defined the type and export it, in the Frontend we could easily create new inputs against this interface.

Reply:
fully agreed. same goes for icon rest props.